### PR TITLE
Feature: Hidden Author Thumbnail

### PR DESCRIPTION
**Pull request by TobiTenno**
### Tenno.tv Pull Request

---

**What I did:**
Added the creator thumbnail if one is provided.
And made the images get hashed, too

---
**Why I did it:**
Xenogelion's request.

---
**Fixes issue (include link):**
N/A

---
**Mockups, screenshots, evidence:**
See deployment.

---

**Was this an issue fix or a suggestion fulfillment?**
Suggestion fulfillment

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -67,10 +67,13 @@ function makeTags(tagArray) {
 }
 function makeRow(video) {
   /* eslint-disable no-plusplus */
+  let authorRow = `<td class="author col-md-2"><a href="https://www.youtube.com/channel/${video.youtube_key}" name="${video.account_name}'s Channel" target="_blank" rel="noopener">`;
+  authorRow += video.author_ty_thumbnail ? `<img class='author-img' alt='${video.account_name}'>` : '';
```
Comment (DrewMasci) [inline assets/js/main.js]: author_yt_thumbnail not ty

Comment (DrewMasci) [reply assets/js/main.js]: Does this image tag have the src attribute for the thumbnail?

**Review by DrewMasci — CHANGES_REQUESTED** [review]
please see comments

File: assets/js/main.js (hunk 1 above)
Comment (TobiTenno) [reply assets/js/main.js]: ah shoot, no it doesn't. fixing!

**Comment by TobiTenno** [comment]
the thumbnail (which i'm assuming is supposed to be a url?) is coming back rather oddly:

```html
<img class="author-img" alt="Xenogelion src=" https:="" yt3.ggpht.com="" -pt7_q07gknu="" aaaaaaaaaai="" aaaaaaaaaqe="" h6hcawsoiis="" s288-mo-c-c0xffffffff-rj-k-no="" photo.jpg'="">
```

hmmm, it's correct in the json, so this is on my end



```json
{
  "video_id": "7cOwUgaeUmI",
  "author_id": "1",
  "is_warframe_content": "1",
  "last_updated": "1527793325",
  "published_on": "1518825600",
  "video_title": "Warframe Tenno Clock #225 Ft. [DE]Connor - &quot;Bring forth the Dark Sectors!&quot;",
  "youtube_key": "UCklKSHzWsDN8T-NZwaWwbOw",
  "account_name": "Xenogelion",
  "author_yt_thumbnail": "https://yt3.ggpht.com/-Pt7_Q07GknU/AAAAAAAAAAI/AAAAAAAAAQE/h6hCAWsOiis/s288-mo-c-c0xffffffff-rj-k-no/photo.jpg",
  "video_tag_ids": {
    "4014": "talk",
    "4015": "machinima"
  }
}
```

**Comment by TobiTenno** [comment]
Looks like browsers are requiring the "titles" attribute now too, i'm adding that.

**Comment by TobiTenno** [comment]
@Xenogelion should be fixed now.

**Pull request merged**